tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1070,7 +1070,7 @@ test_i2c_sht21@90 {
 
 test_i2c_lsm9ds1: lsm9ds1@91 {
 	compatible = "st,lsm9ds1";
-	reg = <0x8e>;
+	reg = <0x92>;
```

Comment:
this should be changed in a separate commit

Reply:
I have a problems with this fast changing file tests/drivers/build_all/sensor/i2c.dtsi, next attempt..